cms/utils/i18n.py

```diff
@@ -77,6 +78,10 @@ def get_current_language():
     It's a replacement for Django's translation.get_language() to make sure the LANGUAGE_CODE will be found in LANGUAGES.
     Overcomes this issue: https://code.djangoproject.com/ticket/9340
     """
+    warnings.warn(
+        "get_current_language() is deprecated; use django.utils.translation.get_language().",
```

Comment:
Do you think it's a good idea to also add stacklevel to the warning.warn so that it shows users where they are using the deprecated code. 

https://www.youtube.com/watch?v=CtFdXBEwYfk

Reply:
Something like:

```python
warnings.warn(
    "get_current_language() is deprecated; use django.utils.translation.get_language().",
    DeprecationWarning, 
    stacklevel=2
)
```

Reply:
Thanks @vinitkumar! That's really useful. Now I understand why you see so many different ways of raising deprecation warnings.